config_templates/gretel/navigator_ft/navigator-ft-tabular.yml

```diff
@@ -0,0 +1,9 @@
+schema_version: "1.0"
```

Comment:
should we rename this file to `default.yml`? I see this pattern but it doesn't seem to be consistent.